submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,115 @@
+
+The other two stop "types" are the Provider and Requester. While one might think that the signals with the Provider Chest and Requester Chest denote which one a stop is, this is not the case. In actuality, a positive resource signal indicates that the Train Stop is a Provider for that resource. An equivalent Negative Signal instead indicates that the Stop is a Requester for that resource. These two signal types instead are simply "thresholds", which are minimums that prevent train from attempting to take from or provide to that stop if the received signals do not meet them. LTN offers the ability to use stack thresholds instead for increased flexibility, although fluids must use the regular threshold signal.
+
+The fact that the sign of the signal is what controls station behavior leads to a common pitfall where an unfamiliar user might find that their requester stop has unintentionally become a provider. The rememdy for this behavior is simply increasing the threshold, either on a per-station basis, or by using the global mod settings. 
+
+These signals are all required to be sent into the lamp part of the hybrid entity which functions as the stop's input port. Another one of the most common errors is sending the control signals to the wrong part of the hybrid entity.
+
+The third part of the entity, the combinator which serves as LTN's output, is what makes exact train loading and multi-item stops possible. Through it, LTN outputs the expected delivery, allowing inserter stack sizes and filters to be configured for the appropriate cargo when multiple items are in play. (The Locked Slots signal should also be sent to the input lamp to ensure excess cargo does not remain in the inserter's hands to contaminate the next train).
+
+One disadvantage to LTN is its learning curve. It can difficult to correctly configure stations as a new player, and it's rather to make mistakes that propagate through your base. Another complication is that LTN uses its own train limit signal, and uses that over vanilla train limits for its requester and its provider stations. However, that signal does not work with its depot stops, which instead require the use of vanilla train limits. (Optera, the mod author, has also stated that there no plans to change that)
+
+### Project Cybersyn - LTN Lite
+
+Named for [Project Cybersyn](https://en.wikipedia.org/wiki/Project_Cybersyn) of Allende's Chile, this mod released just a week after Rail Logistics Dispatcher. Project Cybersyn serves as something of a halfway point between LTN and RLD, removing some of LTN's more features in favor of simplicity.
```

Comment:
> removing some of LTN's more features

"more complex features"?

Reply:
can you shortly elaborate on the removed features?